packages/integration-sdk-dev-tools/config/jest.js

```diff
@@ -18,4 +18,5 @@ module.exports = {
   setupFilesAfterEnv: [
     '<rootDir>/node_modules/@jupiterone/integration-sdk-dev-tools/config/setupJestTestFramework.js',
   ],
+  testTimeout: 20000,
```

Comment:
If this is a limit we're bumping up against in the `graph-aws` project specifically, what if we just override this there? I would definitely prefer to keep the base config as light as possible.

Consider doing something like the following in the target project's `jest.config.js` file:

```diff
- module.exports = require('@jupiterone/integration-sdk-dev-tools/config/jest');
+ module.exports = {
+   ...require('@jupiterone/integration-sdk-dev-tools/config/jest'),
+   testTimeout: 20000,
+ };```